src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.OSX.cs

```diff
@@ -181,6 +181,7 @@ private static unsafe void CreateAndStartRunLoop()
                 }
             }
             s_runLoopThread = new Thread(RunLoopThreadStart);
```

Comment:
Nit: It is not necessary to keep `s_runLoopThread` in a static. It is never used for anything. This can be a local variable.